README.md

```diff
@@ -224,11 +250,17 @@ Here are some of the things coming soon:
   - [x] General system modification commands (writing configuration files, etc)
   - [x] Better log caching and retrieval support
   - [x] Consider showing unified log as output of `pebble run` (use `-v`)
-  - [ ] Add support for automatically removing (double) timestamps from logs
+  - [ ] Automatically restart services that fail
+  - [ ] Support for custom health checks (HTTP, TCP, command)
+  - [ ] Automatically remove (double) timestamps from logs
   - [ ] Improve signal handling, e.g., sending SIGHUP to a service
   - [ ] Terminate all services before exiting run command
   - [ ] More tests for existing CLI commands
 
-## Have fun!
+## Hacking / Development
+
+See [HACKING.md](HACKING.md) for information on how to run and hack on the Pebble codebase during development. In short, use `go run ./cmd/pebble`.
+
+## Contributing
 
-... and enjoy the rest of 2021!
+We welcome quality external contributions. We have good unit tests for much of the code, and a thorough code review process. Please note that unless it's a trivial fix, it's generally worth opening an issue to discuss before submitting a pull request.
```

Comment:
Probably mention Ubuntu CoC and how to sign the CoC https://help.ubuntu.com/community/SigningCodeofConduct

Reply:
I'm not sure signing a CoC should be necessary for submitting a pull request. However, it looks like other Canonical projects ([including snapd](https://github.com/snapcore/snapd/blob/master/CONTRIBUTING.md)) require signing the [Canonical Contributor License Agreement](https://ubuntu.com/legal/contributors) -- I'm guessing we probably want to do the same here, so I've added a link to that.